api/caaCases/index.ts

```diff
@@ -22,7 +33,24 @@ export async function handleCaaCases(req: Request, res: Response, next: NextFunc
   }
 }
 
+export async function handleRoleAssignments(req: Request): Promise<any> {
+  const path = `${getConfigValue(SERVICES_ROLE_ASSIGNMENT_API_PATH)}/am/role-assignments/query`;
+  const payload = {
+    actorId: [req.query.caaCasesFilterValue],
+    roleType: ['CASE']
+  };
+
+  try {
+    const response = await req.http.post(path, payload);
+    return response;
+
+  } catch (error) {
+    console.error('Error: ', error);
```

Comment:
@johnbenjamin-hmcts - should this `console.error()` statement be replaced with `next(error)`?

Reply:
@Daniel-Lam - I referred to MC and for the child method calls we use console.error(), so I used the same.